app/controllers/health/careplans_controller.rb

```diff
@@ -57,7 +57,7 @@ def index
 
     def show
       pdf = careplan_combine_pdf_object
-      file_name = "care_plan_#{@careplan.updated_at.to_s(:db)}"
+      file_name = "care_plan_#{@careplan.updated_at.to_fs(:db)}"
```

Comment:
@ttoomey I'm slightly confused by [this deprecation note](https://guides.rubyonrails.org/7_0_release_notes.html#active-support-deprecations).  Is `to_fs(:db)` acceptable going forward or is there additional cleanup we need to do with this?